copydb/cmd/main.go

```diff
@@ -106,11 +106,13 @@ func main() {
 
 	err = ferry.Initialize()
 	if err != nil {
+		ferry.Ferry.ErrorHandler.ReportError("ferry.initialize", err)
 		errorAndExit(fmt.Sprintf("failed to initialize ferry: %v", err))
 	}
 
 	err = ferry.Start()
 	if err != nil {
+		ferry.Ferry.ErrorHandler.ReportError("ferry.start", err)
```

Comment:
It felt awkard to name this error `ferry.initialize` as well. Lemme know what you folks think.